decisions/Rhovanion.txt

```diff
@@ -612,8 +612,36 @@ political_decisions = {
 					has_pop_religion = men
 					NOT = { has_pop_culture = dorwinrim }
 				}
-				militancy = 1
-				consciousness = 2
+				scaled_consciousness = {
+					factor = 6
+					OR = {
+						ideology = high_nobility
+						ideology = low_nobility
+					}
```

Comment:
Does dropped an OR in here actually work? I've always split it into two scaled_consciousness blocks

Reply:
I was fully confident that it works and didn't check. Gonna test RN.

Reply:
![image](https://github.com/The-Third-Age/TTA-Victoria-2/assets/98686216/025d3dfd-37b3-430e-8b5b-dba38d1548d0)

Oops I messed up brackets